website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -77,3 +77,17 @@
         <p data-bind="html: message, attr.class: messageClass"></p>
     </div>
 </div>
+
+ <script>
+    // Make s3 settings variables globally accessible for relevant JS
+    <%
+        import json
+        from website.addons.s3.settings import DEFAULT_BUCKET_LOCATION
+    %>
+    window.contextVars = $.extend(true, {}, window.contextVars, {
+        s3Settings: {
+            defaultBucketLocationValue: ${json.dumps(DEFAULT_BUCKET_LOCATION['value'])},
```

Comment:
@abought still failing. [Here is my branch](https://github.com/HarryRybacki/osf.io/tree/feature/add_location_based_S3_bucket_creation). To replicate the mako error just go to any project, make sure s3 has credentials for it, and visit the project's settings page.